domain-cc/cc-api-controller/src/main/java/gov/va/vro/api/cc/v3/CCResource.java

```diff
@@ -0,0 +1,24 @@
+package gov.va.vro.api.cc.v3;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import gov.va.vro.api.cc.ResourceException;
+import io.micrometer.core.annotation.Timed;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
```

Comment:
let's use single-class imports here instead of wildcard imports

Reply:
switch to more explicit imports